WTWSMS/decisions/minor_decisions.txt

```diff
@@ -630,7 +637,7 @@ decisions = {
 				factor = 0.001             # Only a small minority of Christians ever go on a pilgrimage
 			}
 			modifier = {
-				factor = 3
+				factor = 50
```

Comment:
I disagree with your opinion. The original number was _one thousandth of one percent_. Multiplying that by fifty still makes it microscopic.